File: documentation/user/en/query/filtering/examples/references/reference-having-any.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		ReferenceHaving("relatedProducts")
        	),
        	Require(
        		EntityFetch(
        			ReferenceContentWithAttributes(
        				"relatedProducts",
        				AttributeContent("category")
        			)
        		)
        	)
        )
	)
);

File: documentation/user/en/query/filtering/examples/references/reference-having-any.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					referenceHaving("relatedProducts")
				),
				require(
					entityFetch(
						referenceContentWithAttributes(
							"relatedProducts",
							attributeContent("category")
						)
					)
				)
			)
		);
	}
);